discord/voice_client.py

```diff
@@ -187,7 +187,7 @@ async def _create_socket(self, server_id, data):
                         'If timeout occurred considering raising the timeout and reconnecting.')
             return
 
-        self.endpoint = endpoint.replace(':80', '')
+        self.endpoint = endpoint.replace(':80', '').replace(':443', '')
```

Comment:
```suggestion
        (self.endpoint, _, _) = endpoint.rpartition(':')
```

I think this breaks if `wss://` ever gets appended but if this happens there are bigger issues so no point in worrying about it.